### PR TITLE
0S2-500 lucyon-it

### PR DESCRIPTION
**Pull request by lucydaniel**
# WELCOME TO DEVOPS-EASY-LEARNING PULL REQUEST SYSTEM 
# Description of the change

> Detailed description here

## Checks and balances

- [ ] Tests added and run
- [ ] (External) Interfaces documented
- [ ] Has security implications (describe below)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)

## Related stories, issues and pulls

> Links to Pivotal Tracker stories, JIRA tickets or other pull requests here

## Security considerations

> Describe any security impact this change may have

### CODE REVIEW

**Comment by lucydaniel** [comment]
new changes

**Comment by devopseasylearning** [comment]
hey looks

File: lucyon-it [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ a good one to
```
Comment (devopseasylearning) [inline lucyon-it:1]: confusing what where you  bout to say?

**Review by devopseasylearning — CHANGES_REQUESTED** [review]

File: lucyon-it [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ a good one to 
+does
+it is lovely
```
Comment (devopseasylearning) [inline lucyon-it:3]: why are saying that?

**Review by devopseasylearning — CHANGES_REQUESTED** [review]

**Pull request closed without merging**